Dockerfile

```diff
@@ -4,29 +4,29 @@ ARG KUBECTL_VERSION=1.22.17
 ARG HELM_VERSION=3.12.1
 ARG KUSTOMIZE_VERSION=5.0.0
 
-FROM golang:1.20-bullseye as kubectl-builder
+FROM golang:1.21-bullseye as kubectl-builder
 ARG KUBECTL_VERSION
 RUN curl -sLf --retry 3 -o kubectl https://storage.googleapis.com/kubernetes-release/release/v${KUBECTL_VERSION}/bin/linux/amd64/kubectl && \
     cp kubectl /usr/local/bin/kubectl && \
     chmod +x /usr/local/bin/kubectl && \
     /usr/local/bin/kubectl version --client=true
 
-FROM golang:1.20-bullseye as helm-builder
+FROM golang:1.21-bullseye as helm-builder
 ARG HELM_VERSION
 RUN curl -sLf --retry 3 -o helm.tar.gz https://get.helm.sh/helm-v${HELM_VERSION}-linux-amd64.tar.gz && \
     mkdir -p helm && tar -C helm -xf helm.tar.gz && \
     cp helm/linux-amd64/helm /usr/local/bin/helm && \
     chmod +x /usr/local/bin/helm && \
     /usr/local/bin/helm version
 
-FROM golang:1.20-bullseye as kustomize-builder
+FROM golang:1.21-bullseye as kustomize-builder
 ARG KUSTOMIZE_VERSION
 RUN curl -sLf --retry 3 -o kustomize.tar.gz https://github.com/kubernetes-sigs/kustomize/releases/download/kustomize%2Fv${KUSTOMIZE_VERSION}/kustomize_v${KUSTOMIZE_VERSION}_linux_amd64.tar.gz \
     && tar -xvzf kustomize.tar.gz -C /usr/local/bin \
     && chmod +x /usr/local/bin/kustomize \
     && /usr/local/bin/kustomize version
 
-FROM golang:1.20-buster as builder
+FROM golang:1.21-buster as builder
```

Comment:
There is no 1.21 buster image

Reply:
After a bit of research it looks like `buster` reached end of life (https://endoflife.software/operating-systems/linux/debian) because `buster` is codename for Debian 10. So I think we can safely change it to bullseye which is Debian 11